package.json

```diff
@@ -22,11 +23,26 @@
   "devDependencies": {
     "@commitlint/cli": "^16.2.3",
     "@commitlint/config-conventional": "^16.2.1",
-    "husky": "^7.0.2"
+    "@readme/eslint-config": "^8.7.3",
+    "caseless": "^0.12.0",
+    "eslint": "^8.14.0",
+    "get-port": "^6.1.2",
+    "husky": "^7.0.2",
+    "jest": "^27.5.1",
+    "prettier": "^2.6.2"
   },
   "commitlint": {
     "extends": [
       "@commitlint/config-conventional"
     ]
-  }
+  },
+  "jest": {
```

Comment:
Noticed this in the Node integration run:

![Screen Shot 2022-04-29 at 10 11 39 AM](https://user-images.githubusercontent.com/33762/165991567-fe89d194-79d6-41b6-8127-e22f0ffc51f1.png)

Can fix this by excluding the `dist` from `modulePathIgnorePatterns`. https://github.com/facebook/jest/issues/8114#issuecomment-475068766

Reply:
I tried adding this to the top level package.json and it didn't work 🤷‍♂️ tried messing about with it for 10/15 mins and couldnt get it to do what I wanted. Gunna leave this for now.